### PR TITLE
Add Device Arris Apex

### PR DESCRIPTION
**Pull request by djamp42**
Tested with Arris APEX-1000

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/arris-apex.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+os: arris-apex
+text: 'Arris Apex'
+type: network
+icon: arris
+mib_dir:
+    - arris
+over:
+    - { graph: '', text: '' }
```
Comment (PipoCanaja) [inline includes/definitions/arris-apex.yaml]: Isn't it any modem counter available that you could poll and display here ? In a CMTS this is usually something you would like to see displayed as the over graph.

**Comment by PipoCanaja** [comment]
Hello @djamp42 
Thanx for your PR. Could you have a look at the Codeclimate issues and solve them ? only a few blanks to remove as far as I can see.
And as I wrote in the YAML file, I am surprised not to see any modem counter. That seems surprising for a CMTS. You can use the "Count" sensor type for this kind of data.

**Comment by djamp42** [comment]
@PipoCanaja This is not a CMTS, it's a IP to QAM Modulator (EdgeQAM) used in cable tv headends.. I'll fix the code climate later tonight.

**Comment by PipoCanaja** [comment]
Other common value in cable operator world is noise. Nothing else to add in that direction ? 
Concerning the tests, unless you add other sensors, should be already good.

**Comment by djamp42** [comment]
No, this is sending out the qam signal so noise isn't measured here, nothing in the upstream direction either on this thing either. I looked to tried to find something for the hover graphs but nothing really made sense.

**Pull request merged**